gene_expression_processing.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
+
+
+"""
+Hail Batch workflow to create gene expression files.
+This script will:
+
+- select genes to test based on expression (per cell type)
+- build chromosome & cell type specific phenotype covariate files
+- use gene info to create cis-window files
+- export pheno_cov files as tsv files
+- export cis_window files as tsv files
+
+More details in README
+output files in tob_wgs_genetics/saige_qtl/input
+"""
+
+import os
+import logging
+import math
+import sys
+
+import click
+import hail as hl
+import hailtop.batch as hb
+import pandas as pd
+import scanpy as sc
+
+from cpg_utils import to_path
+from cpg_utils.hail_batch import copy_common_env, dataset_path, output_path
+from cpg_workflows.batch import get_batch
```

Comment:
Is there a reason you're using the cpg_workflows get_batch command?
@MattWellie, is this something we should move to cpg-utils?

Reply:
Yup, happy to look into that